src/components/footer/index.jsx

```diff
@@ -16,36 +15,25 @@ import { useResource } from '../../lib/use-resource';
  * }
  * const repos = await api('/orgs/preactjs/repos?per_page=100');
  * const list = new Set((await Promise.all(repos.map(r => getContribs(r.owner.login, r.name)))).flat().filter(n => !n.endsWith('-bot') && !n.endsWith('[bot]')));
- * copy(JSON.stringify(list, null, 2));
+ * copy(JSON.stringify([...list], null, 2));
  *
  * And paste the results into src/assets/contributors.json
  */
 
 /**
  * Display a random contributor of the list above.
- * @param {any[]} deps
  */
-function useContributors(deps) {
+function useContributors() {
 	const contributors = useResource(() =>
 		fetch('/contributors.json').then(r => r.json()),
 		['/contributors.json']
 	);
 
-	const [value, setValue] = useState(
-		contributors ? contributors[new Date().getMonth()] : undefined
-	);
-
-	useEffect(() => {
-		if (contributors)
-			setValue(contributors[(Math.random() * (contributors.length - 1)) | 0]);
-	}, [...deps]);
-
-	return value;
```

Comment:
We don't really need all this, picking a new contrib every time the footer rerenders is fine.